### PR TITLE
Fix pugixml whitespace parsing in instances

### PR DESCRIPTION
**Pull request by rschili**
properties that contain only whitespaces were not parsed correctly as pugi considers the whitespace insignificant. An option can be used to change the default behavior.

### CODE REVIEW

File: iModelCore/ecobjects/src/Format.cpp [hunk 1]
```diff
@@ -312,7 +312,7 @@ SchemaReadStatus ECFormat::ReadXml(pugi::xml_node unitFormatNode, ECSchemaReadCo
 
     SetNumericSpec(spec);
     pugi::xml_node child = unitFormatNode.first_child();
-    if (child)
+    if (child && child.type() == pugi::xml_node_type::node_element)
```
Comment (ColinKerr) [inline iModelCore/ecobjects/src/Format.cpp:315]: Do we need some more tests to ensure that we have updated the deserialization code to take these new empty string nodes into account?

Comment (rschili) [reply iModelCore/ecobjects/src/Format.cpp:315]: There is an existing test that covers the problem, but it's not the intended purpose of that test. I'll add some dedicated testing for this

**Review by ColinKerr — APPROVED** [review]
